docs/source/api/references/graphstorm.config.rst

```diff
@@ -0,0 +1,20 @@
+.. _apiconfig:
+
+graphstorm.config
+===================
+
+GraphStorm model training and inference CLIs have a set of :ref:`built-in configurations <configurations-run>`.
+These configurations are defined and managed in ``graphstorm.config`` module whose ``get_argument_parser()``
+method can help users to read in these built-in configurations either from a yaml file or from CLI arguments.
+This ``get_argument_parser()`` is useful when users want to convert your customized models to use GraphStorm CLIs.
+
+.. currentmodule:: graphstorm.config
```

Comment:
Shall we add a code example here to demonstrate how to use this function?

Reply:
The code example is in the `get_argument_parser` own doc string.